### PR TITLE
Apply custom secondary color to Pages list

### PR DESCRIPTION
**Pull request by gergo85**
Before:
before

After:
after

### CODE REVIEW

**Comment by w20k** [comment]
@gergo85 could you rebuild assets.

File: modules/backend/models/brandsetting/custom.less [hunk 1]
```diff
@@ -81,7 +89,7 @@ table.table.data {
     tbody {
         tr.active td {
             &:first-child {
-                border-left: 3px solid @brand-secondary;
+                border-left-color: @brand-secondary;
```
Comment (w20k) [inline modules/backend/models/brandsetting/custom.less]: @	gergo85 why did you remove `3px border`?

Comment (gergo85) [reply modules/backend/models/brandsetting/custom.less]: The custom.less file only contains color definition, is not?

Comment (w20k) [reply modules/backend/models/brandsetting/custom.less]: Mainly - yes, but just removing the border. If it's a part of the ticket, please provide before/after example!

There is a logo section beside colors.
```
//
// Logos
//

.oc-logo-transparent when not (@logo-image = '') {
    background-image: url('@{logo-image}') !important;
}

.oc-logo when not (@logo-image = '') {
    background-image: url('@{logo-image}');
}
```

**Comment by gergo85** [comment]
How to rebuild assets?

**Comment by gergo85** [comment]
I restored the border.

**Comment by w20k** [comment]
@gergo85  `php artisan october:util compile less` 
Thanks!

**Comment by gergo85** [comment]
@w20k I think, no need to rebuild assets. The custom settings display via inline css.

**Comment by w20k** [comment]
You're right @gergo85 thanks!
